dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl

```diff
@@ -113,4 +113,11 @@ namespace Microsoft.Windows.ApplicationModel.Resources
         static String TargetSize { get; };
         static String Theme { get; };
     }
-} // namespace Microsoft.Windows.ApplicationModel.Resources
+
+    [contract(MrtCoreContract, 1)]
+    runtimeclass ApplicationLanguages
```

Comment:
New API, so please include a markdown file for API review.

Reply:
Often better to make a PR with just a markdown spec for review, separate from implementation/test/etc. Streamlines the API Review process for everyone involved

Reply:
this is in progress, thanks for pointing out